### PR TITLE
Contributor Title tooltip

### LINKED ISSUE #9400: Contributor Title Tooltip
You should be able to hover over a contributor’s name tag in chat and see their contributor title (for example,  "Friend Artisan").

### PR DESCRIPTION
**Pull request by librarianmage**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Your_Local_Install_to_Modify_Habitica%27s_Website_and_API for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #9400 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

[//]: # (Put User ID in here - found on the Habitica website at User Icon > Settings > API)

----
UUID:

### CODE REVIEW

File: website/client/components/chat/chatMessages.vue [hunk 1]
```diff
@@ -24,11 +24,12 @@
         .message-hidden(v-if='msg.flagCount > 1 && user.contributor.admin') Message hidden
         .card-block
             h3.leader(
-              :class='userLevelStyle(cachedProfileData[msg.uuid])'
+              :class='userLevelStyle(msg)'
```
Comment (paglias) [inline website/client/components/chat/chatMessages.vue]: why was this changed?

Comment (librarianmage) [reply website/client/components/chat/chatMessages.vue]: In the above PR, @Alys requested that instead of using the cached profile for contributor info, that I use the information stored in the chat message itself.

Comment (paglias) [reply website/client/components/chat/chatMessages.vue]: you're right, sorry!

File: website/client/components/chat/chatMessages.vue [hunk 2]
```diff
@@ -268,6 +270,7 @@ import tier7 from 'assets/svg/tier-7.svg';
 import tier8 from 'assets/svg/tier-mod.svg';
 import tier9 from 'assets/svg/tier-staff.svg';
 import tier10 from 'assets/svg/tier-npc.svg';
+import bTooltip from 'bootstrap-vue/lib/directives/tooltip';
```
Comment (paglias) [inline website/client/components/chat/chatMessages.vue]: hey just noticed this, with the new version of bootstrap-vue we're using it's not necessary to import directive and components manually, can you remove the import statement and this line https://github.com/HabitRPG/habitica/pull/9413/files#diff-e01bfe286bce1620d57ddaa25f228043R285?

**Review by paglias — DISMISSED** [review]

**Comment by librarianmage** [comment]
@paglias Fixed!

**Comment by paglias** [comment]
@MathWhiz I think the tooltip is not working, the code uses contributor.title but the title is under a different property

schermata 2017-11-16 alle 16 31 04

**Comment by librarianmage** [comment]
@paglias changed title to text

**Comment by paglias** [comment]
@MathWhiz one last issue!

The name element occupies the entire width of the box so the popover appears weirdly in the middle

schermata 2017-11-16 alle 16 35 17

schermata 2017-11-16 alle 16 35 25

**Comment by librarianmage** [comment]
@paglias Does this work? I'm not able to test it right now.

**Comment by paglias** [comment]
@MathWhiz it still doesn't work, but you can revert the last commit, add `display: inline-block` to the h3 and it'll work

**Comment by librarianmage** [comment]
@paglias Updated!
